makeicons/pages.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+This file is just the bok_choy pages file for our icons. bok_choy
+is a test framework, so we follow this split into pages/icon, although
+it is a little artifical in this case.
+"""
+from bok_choy.page_object import PageObject
+
+
+class IconsPage(PageObject):
```

Comment:
This has nothing to do with tests. This is code which will render .png icons for use in the XBlock. I made pretty icons in HTML+CSS+Unicode. These do not render correctly on Mac. This is fundamental. Ubuntu has plain smiley faces, while Mac has 3d rendered yellow smiley faces. Those looked horrible, and @explorerleslie (correctly) blocked from shipping the XBlock as is.

bok choy is the easiest way to go .html → .png. 

I will add better documentation. 

I do not currently have a way to do bok choy tests for the XBlock itself. @benpatterson was planning to look into this, but has not done so yet.